critiquebrainz/data/model/review.py

```diff
@@ -180,6 +181,11 @@ def list(cls, **kwargs):
         if user_id is not None:
             query = query.filter(Review.user_id == user_id)
 
+        exclude = kwargs.pop('exclude', None)
+        if exclude is not None:
+            for review_id in exclude:
+                query = query.filter(Review.id != review_id)
```

Comment:
Isn't it better to use "NOT IN" with a list instead? https://www.postgresql.org/docs/9.6/static/functions-comparisons.html